sandbox/cri/config.toml

```diff
@@ -0,0 +1,189 @@
+# Firecracker
+[proxy_plugins]
+  [proxy_plugins.firecracker-naive]
+    type = "snapshot"
+    address = "/var/run/firecracker-containerd/snapshotter.sock"
+  [proxy_plugins.firecracker-dm-snapshotter]
+    type = "snapshot"
+    address = "/var/run/firecracker-containerd/snapshotter.sock"
+
+# The "plugins.cri" table contains all of the server options.
+[plugins.cri]
+
+  # stream_server_address is the ip address streaming server is listening on.
+  stream_server_address = "127.0.0.1"
+
+  # stream_server_port is the port streaming server is listening on.
+  stream_server_port = "0"
+
+  # enable_selinux indicates to enable the selinux support.
+  enable_selinux = false
+
+  # sandbox_image is the image used by sandbox container.
+  sandbox_image = "k8s.gcr.io/pause:3.1"
+
+  # stats_collect_period is the period (in seconds) of snapshots stats collection.
+  stats_collect_period = 10
+
+  # systemd_cgroup enables systemd cgroup support. This only works for runtime
+  # type "io.containerd.runtime.v1.linux".
+  # DEPRECATED: use Runtime.Options for runtime specific config for shim v2 runtimes.
+  #   For runtime "io.containerd.runc.v1", use the option `SystemdCgroup`.
+  systemd_cgroup = false
+
+  # enable_tls_streaming enables the TLS streaming support.
+  # It generates a self-sign certificate unless the following x509_key_pair_streaming are both set.
+  enable_tls_streaming = false
+
+  # "plugins.cri.x509_key_pair_streaming" contains a x509 valid key pair to stream with tls.
+  [plugins.cri.x509_key_pair_streaming]
+    # tls_cert_file is the filepath to the certificate paired with the "tls_key_file"
+    tls_cert_file = ""
+
+    # tls_key_file is the filepath to the private key paired with the "tls_cert_file"
+    tls_key_file = ""
+
+  # max_container_log_line_size is the maximum log line size in bytes for a container.
+  # Log line longer than the limit will be split into multiple lines. -1 means no
+  # limit.
+  max_container_log_line_size = 16384
+
+  # disable_cgroup indicates to disable the cgroup support.
+  # This is useful when the daemon does not have permission to access cgroup.
+  disable_cgroup = false
+
+  # disable_apparmor indicates to disable the apparmor support.
+  # This is useful when the daemon does not have permission to access apparmor.
+  disable_apparmor = false
+
+  # restrict_oom_score_adj indicates to limit the lower bound of OOMScoreAdj to
+  # the containerd's current OOMScoreAdj.
+  # This is useful when the containerd does not have permission to decrease OOMScoreAdj.
+  restrict_oom_score_adj = false
+
+  # "plugins.cri.containerd" contains config related to containerd
+  [plugins.cri.containerd]
+
+    # snapshotter is the snapshotter used by containerd.
+    snapshotter = "firecracker-naive"
```

Comment:
Are you planning to test with both firecracker-naive and firecracker-dm?

Reply:
I removed the devmapper and will only test against naive, for now at least.